var/spack/repos/builtin/packages/qthreads/package.py

```diff
@@ -85,3 +85,58 @@ def configure_args(self):
                     % self.spec.variants['scheduler'].value)
 
         return args
+
```

Comment:
FWIW. From a static analysis standpoint, there should be only 1 line between the `return` and the properties.